src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
-}
+  const DayRentCost = 40;
+  const ThreeDayBonus = 20;
+  const FiveDayBonus = 50;
```

Comment:
don't use capitalized names
```suggestion
  const dayRentCost = 40;
  const threeDayBonus = 20;
  const fiveDayBonus = 50;
```